plugin-packs/postcss-preset-env/src/lib/get-unsupported-browsers-by-feature.js

```diff
@@ -1,26 +1,41 @@
-import * as caniuse from 'caniuse-lite';
+const browsers = [
+	'ie',
+	'edge',
+	'firefox',
+	'chrome',
+	'safari',
+	'opera',
+	'ios_saf',
+	'android',
+	'op_mob',
+	'and_chr',
+	'and_ff',
+	'and_uc',
+	'samsung',
+	'and_qq',
+	'baidu',
+	'kaios',
+];
 
 // return a list of browsers that do not support the feature
 export default function getUnsupportedBrowsersByFeature(feature) {
-	const caniuseFeature = caniuse.features[feature];
-
 	// if feature support can be determined
-	if (caniuseFeature) {
-		const stats = caniuse.feature(caniuseFeature).stats;
+	if (feature.browser_support) {
+		const query = [];
+
+		browsers.forEach(browser => {
+			const browserSupport = feature.browser_support[browser];
+			// If the version is something like TP we can't do < TP
```

Comment:
Are pre-release versions counted as vendor implementations in cssdb?

Reply:
Yes they are, which is not right. Will fix that, thanks for flagging!

Reply:
This has been fixed, only affected `:has`.

Reply:
Nice!